system-parachains/bridge-hubs/bridge-hub-kusama/src/bridge_to_ethereum_config.rs

```diff
@@ -25,3 +25,42 @@ pub type SnowbridgeExporter = EthereumBlobExporter<
 	snowbridge_pallet_outbound_queue::Pallet<Runtime>,
 	snowbridge_core::AgentIdOf,
 >;
+
+#[cfg(feature = "runtime-benchmarks")]
+pub mod benchmark_helpers {
+	use crate::{EthereumBeaconClient, Runtime, RuntimeOrigin};
```

Comment:
```suggestion
	use crate::{EthereumBeaconClient, EthereumGatewayAddress, Runtime, RuntimeOrigin};
```

Reply:
I moved things around, please let me know if it looks OK.